.ci/azure/linux.yml

```diff
@@ -99,3 +111,23 @@ jobs:
     env:
       LD_LIBRARY_PATH: $(BIN_DIR)/lib
     displayName: 'Java tests'
+
+  - script: |
+      sudo apt-get install -y python3-setuptools
+      python3 -m pip install \
+          defusedxml>=0.5.0 \
+          networkx>=1.11 \
+          test-generator==0.1.1 \
+          torch==1.7.0 \
+          torchvision==0.8.1 \
+          opencv-python==4.4.0.46
+
+      git clone --depth 1 -b v0.3 https://github.com/facebookresearch/detectron2
+      python3 -m pip install -e detectron2
+
+      python3 test.py
```

Comment:
please separate installation stage and actual tests execution to different steps. This will improve pipeline analytics

Reply:
Thanks! Done